### PR TITLE
Track saved version per hour, day, week, month, year, and number of times a version with parameter

### PR DESCRIPTION
**Pull request by softwareguru90**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -291,6 +291,8 @@ COOKIE_MAX_AGE      | session cookie max age, defaults to 90 days; can be set to
 HOME_PORT           | port number to listen on for REST API server; if set to "share", add API endpoints to regular grist port.
 PORT                | port number to listen on for Grist server
 REDIS_URL           | optional redis server for browser sessions and db query caching
+GRIST_SNAPSHOT_TIME_CAP           | optional. Track saved version per hour, day, week, month, year, and number of times a version  Usage: {"hour": 25, "day": 32, "isoWeek": 12, "month": 96, "year": 1000}
```
Comment (paulfitz) [inline README.md:294]: Is there something missing in the text, around `and number of times a version  Usage`?

File: README.md [hunk 2]
```diff
@@ -291,6 +291,8 @@ COOKIE_MAX_AGE      | session cookie max age, defaults to 90 days; can be set to
...
+GRIST_SNAPSHOT_KEEP           | optional. How many Snapshot versions should be kept
```
Comment (paulfitz) [inline README.md:295]: Feels a bit ambiguous. Perhaps `Number of recent snapshots to retain unconditionally for a document, regardless of when they were made`

**Review by paulfitz — COMMENTED** [review]
All the code looks good now, thanks - just some comments on the README description.

File: README.md (hunk 1 above)
Comment (softwareguru90) [reply README.md]: Explaining something is really hard for a developer. Sorry for that. I just write something like below: 

```
Define the caps for tracking buckets. Usage: {"hour": 25, "day": 32, "isoWeek": 12, "month": 96, "year": 1000}
```

What do you think? Is it more clear?

**Review by paulfitz — APPROVED** [review]
Thanks @softwareguru90 !

**Pull request merged**